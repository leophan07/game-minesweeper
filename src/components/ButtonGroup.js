import styled from 'styled-components';

export default styled.div`
  margin: 20px auto;
  display: flex;
  flex-direction: row;
  justify-content: space-evenly;
`;