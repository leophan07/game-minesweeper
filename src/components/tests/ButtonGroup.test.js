import React from 'react';
import { render } from '@testing-library/react';

import ButtonGroup from '../ButtonGroup';

describe('<ButtonGroup />', () => {
  it('should render an <div> tag', () => {
    const { container } = render(<ButtonGroup />);
    expect(container.querySelector('div')).toBeTruthy();
  });

  it('should match the snapshot', () => {
    const { container } = render(<ButtonGroup />);
    expect(container).toMatchSnapshot();
  });
});
