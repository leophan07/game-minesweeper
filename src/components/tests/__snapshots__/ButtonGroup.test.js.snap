// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonGroup /> should match the snapshot 1`] = `
<div>
  <div
    class="sc-bdVaJa hWdoSm"
  />
</div>
`;
